app/main/appium.js

```diff
@@ -150,9 +147,8 @@ export function createNewSessionWindow (win) {
   });
 
   // When you close the session window, kill its associated Appium session (if there is one)
-  let sessionID = sessionWin.webContents.id;
   sessionWin.on('closed', () => {
-    killSession(sessionID);
+    // TODO W2D use IPC to trigger a delete session in renderer
```

Comment:
should the TODO stay?

Reply:
Ah I missed that, will add the functionality before merging

Reply:
ok, added session deletion on window close in the renderer